tests/legacy-cli/e2e/tests/misc/completion.ts

```diff
@@ -2,49 +2,61 @@ import { execAndWaitForOutputToMatch } from '../../utils/process';
 
 export default async function () {
```

Comment:
Nit: Should these commits be squashed or swapped in ordering? As is, if the second commit got rolled back, CI would fail at just the first commit.